starwars/src/App.js

```diff
@@ -9,10 +33,34 @@ const App = () => {
   // side effect in a component, you want to think about which state and/or props it should
   // sync up with, if any.
 
+  const [charactersDataState, setCharactersDataState] = useState([]);
+ 
+  useEffect(() => {
+    axios.get(`https://swapi.co/api/people`)
+      .then(response => {
+          setCharactersDataState(response.data.results)
+        })
+
+      .catch(error => {
+        console.log(error)
+      })
```

Comment:
A better way to handle error is to create another slice of state, then render the error in the view
e.g
```
const [error, setError] = useState(null);
```
then in the catch block
```
 .catch(error => {
        setError(error.message);
      })
```
then in the jsx:
```
{ error && <p>{error}</p>